### PR TITLE
docs: add Woodpecker CI integrations example

### PR DESCRIPTION
**Pull request by 6543**
## Description

Add pipeline example how to integrate with Woodpecker CI.

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] ~~I've added tests that prove my fix is effective or that my feature works.~~ NaN
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] ~~I've added usage information (if the PR introduces new options)~~ NaN
- [ ] ~~I've included a "before" and "after" example to the description (if the PR is a user interface change).~~ NaN

### CODE REVIEW

**Comment by 6543** [comment]
image

**Comment by 6543** [comment]
hmm `integrations` is not a valid scope, what is suggested?

File: docs/docs/integrations/woodpecker-ci.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+pipeline:
+  securitycheck:
+    group: test
+    image: aquasec/trivy:latest
+    commands:
+      - trivy fs --exit-code 0 --skip-dirs web/ --skip-dirs docs/ --severity UNKNOWN,LOW .
+      - trivy fs --exit-code 1 --skip-dirs web/ --skip-dirs docs/ --severity MEDIUM,HIGH,CRITICAL .
+```
+
+Woodpecker does use trivy itsleve so you can see an [Example][example] run at it's [Repository][repository] and how it was [added](https://github.com/woodpecker-ci/woodpecker/pull/1163).
```
Comment (seabass-labrax) [inline docs/docs/integrations/woodpecker-ci.md:15]:
```suggestion
Woodpecker does use Trivy itself so you can see an [Example][example] run at its [Repository][repository] and how it was [added](https://github.com/woodpecker-ci/woodpecker/pull/1163).
```

**Review by seabass-labrax — CHANGES_REQUESTED** [review]
Would love to see this added to the docs :smiley:

**Review by seabass-labrax — APPROVED** [review]
LGTM!

File: docs/docs/integrations/woodpecker-ci.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Woodpecker CI
+
+This is a simple example configuration `.woodpecker/trivy.yml` that shows how you could get started:
+
+```yml
+pipeline:
+  securitycheck:
+    group: test
```
Comment (6543) [inline docs/docs/integrations/woodpecker-ci.md:8]:
```suggestion
```
group not needed for single step pipeline ...

Comment (6543) [reply docs/docs/integrations/woodpecker-ci.md:8]: ... and make it simpler too

**Comment by 6543** [comment]
@knqyf263 is anything missing?

Should I squash?

**Comment by 6543** [comment]
@knqyf263 do you have any update?

**Comment by knqyf263** [comment]
@AnaisUrlichs Could you take a look? I'm not sure where we should put it. Tutorial?

File: docs/docs/integrations/woodpecker-ci.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+This is a simple example configuration `.woodpecker/trivy.yml` that shows how you could get started:
+
+```yml
+pipeline:
+  securitycheck:
+    image: aquasec/trivy:latest
+    commands:
+      - trivy fs --exit-code 0 --skip-dirs web/ --skip-dirs docs/ --severity UNKNOWN,LOW .
+      - trivy fs --exit-code 1 --skip-dirs web/ --skip-dirs docs/ --severity MEDIUM,HIGH,CRITICAL .
```
Comment (AnaisUrlichs) [inline docs/docs/integrations/woodpecker-ci.md:11]: Is there a reason that these commands have to be so "comprehensive"? I would like to just have a simple example and then mention below that the command can be replaced with any trivy command. What do you think?

**Comment by AnaisUrlichs** [comment]
Great! Thank you for adding it to the docs! I left one minor comment -- otherwise, yes, I think part of the tutorials between the other CI examples, it's good @knqyf263

File: docs/docs/integrations/woodpecker-ci.md (hunk 3 above)
Comment (6543) [reply docs/docs/integrations/woodpecker-ci.md:11]: well what about delete L10 and add a comment about `# use any trivy command, if exit code is 0 woodpecker marks it as passed, else it assumes it failed` ?

**Comment by 6543** [comment]
@AnaisUrlichs to https://github.com/aquasecurity/trivy/pull/2823#discussion_r985251887 Yes / No ?

**Comment by 6543** [comment]
@AnaisUrlichs done :rocket:

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**